fluentd/Dockerfile

```diff
@@ -36,6 +36,11 @@ ADD vendored_gem_src/ ${HOME}/vendored_gem_src/
 ADD install-gems.sh *.patch.sh *.patch ${HOME}/vendored_gem_src/
 
 RUN cd ${HOME}/vendored_gem_src/ && ./install-gems.sh && cd / && rm -rf ${HOME}/vendored_gem_src/
+RUN mkdir -p /tmp/gems
+ADD lib/fluentd-okd-config-generator /tmp/gems/fluentd-okd-config-generator/
+RUN cd /tmp/gems/fluentd-okd-config-generator && \
```

Comment:
I would advise against creating and installing a gem for this.  Better to copy the files into `/etc/fluent/plugin` instead.

Reply:
It's not a fluent plugin but a utility that is outputs conf files.  What are the reservations given we are vendoring everything and it lives only within our flunt image?

Reply:
There's no reason to create a gem - it's just extra work - just use the ruby files directly

Reply:
We also have other utilities that output files - we don't have gems for those.